packages/protocol/test/TestBase.sol

```diff
@@ -51,6 +54,22 @@ abstract contract TestBase is Test {
     address internal SGX_Z = getRandomAddress();
 }
 
+library LibBridgedTokenDeployer {
+    function deployLogicContract(
```

Comment:
can we deploy the ERC token implementation contract the same way as any other contract without using Create2?

Reply:
A simple 'new' makes it for sure , just re-used the previous approach if we ever wanted to have deterministic deployments. But i can modify it.

Reply:
I don't think we need deterministic deployments unless I missed something. Only the proxy needs to be deterministic per token.

Reply:
Actually maybe we do have a problem here, maybe not a big one:

currently the btoken address is not deterministic.

Reply:
I prepare a draft PR for you @adaki2004

Reply:
Yes, i agree. We only do care about 1 implementation (which will be shared) and different proxies. I think it has nothing to do with `CREATE2` - only in case we want to have some addresses deterministic.